dotCMS/src/main/java/com/dotmarketing/common/reindex/ReindexThread.java

```diff
@@ -225,6 +225,7 @@ private void startProcessing(int sleep, int delay) {
 	 * containing the content identifier will be sent to the user via the
 	 * Notifications API to take care of the problem as soon as possible.
 	 */
+	@CloseDBIfOpened
```

Comment:
Not pretty sure if the thread is saving something on the db, but if it is should be WrapInTransaction

Reply:
let's undo this one, since the old closeConnection is inside a `while` loop